### PR TITLE
Shop by Category Widgets update

### PR DESCRIPTION
**Pull request by Souvik-Ignitiv**
Shop by Category Widgets dynamic category from builder io

### CODE REVIEW

File: cms/components/CmsShopByCategory/CmsShopByCategory.styles.ts [hunk 1]
```diff
@@ -27,7 +30,8 @@ export const ShopByCategoryStyle = {
     width: '100%',
   },
   categoryItem: {
-    background: '#dddddd',
+    background: '#e8ddfe',
```
Comment (bsatignitiv) [inline cms/components/CmsShopByCategory/CmsShopByCategory.styles.ts]: Add color code to the theme file and use theme to provide color

File: cms/components/CmsShopByCategory/CmsShopByCategory.tsx [hunk 2]
```diff
@@ -107,16 +114,28 @@ const CmsHomePageCategory = (props: HomePageProps) => {
...
+                sx={ShopByCategoryStyle.categoryLink}
+              >
+                <Box sx={ShopByCategoryStyle.categoryItemWrapper}>
+                  <Box sx={ShopByCategoryStyle.categoryImageWrapper}>
+                    <Box
+                      component="img"
+                      src={productGetters.handleProtocolRelativeUrl(
+                        categorySearchGetters.getCoverImage(item)
+                      )}
+                      // alt={item.imageAlt}
```
Comment (bsatignitiv) [inline cms/components/CmsShopByCategory/CmsShopByCategory.tsx]: Remove comment

**Review by bsatignitiv — CHANGES_REQUESTED** [review]

File: cms/components/CmsShopByCategory/CmsShopByCategory.styles.ts (hunk 1 above)
Comment (Souvik-Ignitiv) [reply cms/components/CmsShopByCategory/CmsShopByCategory.styles.ts]: resolves as per comment and color is coming from theme file

File: cms/components/CmsShopByCategory/CmsShopByCategory.tsx (hunk 2 above)
Comment (Souvik-Ignitiv) [reply cms/components/CmsShopByCategory/CmsShopByCategory.tsx]: resolves as per comment remove commented code

**Pull request merged**